docs/framework/social-post-generator.mdx

```diff
@@ -2,218 +2,448 @@
 title: "Social post generator"
 ---
 
-In this tutorial, you’ll use the Writer Framework to build an application that generates social media posts and tags based on user input.
+In this tutorial, you'll use the Writer Framework to build an AI-powered tool for generating social media posts and tags based on the input you provide!
+
+The process will take only minutes using a drag-and-drop visual editor to build the user interface and Python for the back-end code.
+
+Here's what the finished project will look like:
 
 ![Finished social post generator application](/framework/images/tutorial/social_post/sp_gen_1.png)
 
 
+## Prerequisites
+
+Before starting, ensure you have:
+
+- **A Writer account:** You'll need an account to use Writer Framework, and [you can sign up for a free one](https://app.writer.com/aistudio/signup).
```

Comment:
>You'll need an account to use Writer Framework

Can we change this? In fact, we should start emphasizing everywhere that you don't need a Writer account to use Writer Framework, only to use Writer's AI features which are accessible via the Framework.

Reply:
Ah good catch @ramedina86 -- yes @AccordionGuy let's say something like "You'll need an account to use the AI module in Writer Framework"

Reply:
Updated!